apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/SourceApiTestConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2;
+
+import io.stargate.sgv2.api.common.grpc.SourceApi;
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.Produces;
+
+/** Test config for the source api, so common can boot itself. */
+@ApplicationScoped
+public class SourceApiTestConfiguration {
+
+  @Produces
+  @SourceApi
+  public String sourceApi() {
+    return "rest";
```

Comment:
For tests could perhaps use a distinct value, separate from values we use in prod?
Unless we have validation settings that prevent use of id like "test" in which case never mind.

Reply:
But what would we gain with this? This was made so we can start the commons context in the tests, basically it's not used nor asserted anywhere..

Reply:
I was thinking it'd avoid test accidentally passing. If and when not tested yeah never mind.

And since we'd need matching Enum (which I did not realize when adding this comment) not worth it either.